auth0/auth0-cli
# Upload properly edited user import template

### Description

Fixing `auth0 user import`.

When using the `auth0 users import` command, after choosing a template, it will then open that template in an editor.  
Instead of uploading the edited template, it will instead upload the default template, ignoring any changes made in the editor. 

This pull request fixes this functionality to upload the template saved after the editor closes. 

### Testing

1. Run `auth0 users import`
2. Choose a db connection.
3. Choose a template.
4. Make some changes to the template in the editor.
5. Import template
6. Check to make sure that the template being uploaded is the one saved from the editor.
